docs/mkcloud.md

```diff
@@ -20,6 +20,14 @@ or testing purposes.
   $ sudo service libvirtd start  # to start the daemon
   ```
 
+  If you are using `libvirtd >= 1.3.0` then you should also start the
+  `virtlogd` service.
+
+  ```
+  $ sudo service virtlogd status # to check the status
+  $ sudo service virtlogd start  # to start the daemon
```

Comment:
@bmwiedemann from what I can tell changes in the `libvirtd` service (stop, restart etc) do not affect the `virtlogd` one. The two service (or socket) units do not have any sort of dependency on one another and they act independently.